### PR TITLE
Fix URL for component-search endpoint

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
cc @samuzzal-choudhury @tisnik 

Just an interim solution until we get API token for CI environment.

**Comment by msrb** [comment]
related: https://github.com/fabric8-analytics/fabric8-analytics-common/issues/158

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ node('docker') {
             dir('integration-tests') {
                 timeout(10) {
                     withEnv(["F8A_API_URL=${apiUrl}", "F8A_JOB_API_URL=${jobsApiUrl}", "F8A_ANITYA_API_URL=${anityaApiUrl}"]) {
-                        sh './runtest.sh --tags=-jobs.requires_auth'
+                        sh './runtest.sh --tags=-jobs.requires_auth --tags=-api.requires_auth'
```
Comment (tisnik) [inline Jenkinsfile]: just being curious why the tag is used twice?

**Comment by msrb** [comment]
Do not merge please, I am gonna drop the last two commits when https://github.com/fabric8-analytics/fabric8-analytics-common/pull/164 is ready and merged.

**Comment by msrb** [comment]
Ready.

**Comment by msrb** [comment]
Merged with https://github.com/fabric8-analytics/fabric8-analytics-common/pull/164, closing this one.

**Pull request closed without merging**